pages/user/posts.tsx

```diff
@@ -0,0 +1,293 @@
+import { Transition } from "@headlessui/react";
```

Comment:
Probably need to code-split this, this is doing too much.

Reply:
But i will pass it for now, moving on to the next one

Reply:
Okay!